static/docs/use-cases/shared-storage-on-nfs.md

```diff
@@ -0,0 +1,148 @@
+your shared directory is owned by that group and has respective permissions.
+
+Let's create a mount point of client side.
+
+```dvc
+$ mkdir -p /mnt/dataset/
+```
+
+From `/mnt/dataset/` you will be able to access `/storage` directory present in
+host server from your local machine.
+
+## Configuring Cache location
+
+After mounting the shared directory on client side. Assuming project code is
+present in `/project`. Let's initialize a `dvc repo`.
+
+```dvc
+$ cd /project/
+$ git init
+$ dvc init
+$ git add .dvc .gitignore
+$ git commit . -m "initialize DVC"
+```
+
+With `dvc init`, we initialized a DVC repository. For more information, visit
+[here](/doc/get-started/initialize).
+
+**Tell DVC to use the directory we've set up as an external cache location by
+running:**
+
+```dvc
+$ dvc cache dir /mnt/dataset/storage
+```
+
+`dvc cache dir /path/to/cache/directory` - sets cache directory location.
+
+```dvc
+$ dvc config cache.type "reflink,symlink,hardlink,copy"
+```
+
+`cache.type "reflink,symlink,hardlink,copy"` - link type that DVC should use to
+link data files from cache to your workspace. It enables symlinks to avoid
+copying large files. For more information, vist
+[here](/doc/user-guide/large-dataset-optimization).
+
+```dvc
+$ dvc config cache.protected true
+```
+
+`cache.protected true` - to make links `read only` so that we you don't corrupt
+data accidentally present in the workspace. Since, we are using `symlinks`
+between the cache and local workspace because both are located on different
+filesystem.
+
+Also, let Git know about the changes we have done.
+
+```dvc
+$ git add .dvc .gitignore
+$ git commit . -m "DVC cache location updated"
+```
+
+## Add data to DVC cache
+
+Now, add first version of the dataset into the DVC cache (this is done once for
+a dataset).
+
+```dvc
+$ cd /mnt/dataset/
+$ cp -r . /project/
+$ cd /project
+$ mv /mnt/dataset/project_data/ data/
+$ dvc add data
+```
+
+After copying the data, we have moved the data that is present in the
+`/mnt/dataset/project_data/` to `./data` directory. This is only done once for a
+dataset.
+
+`dvc add data` will take files in `data` directory under DVC control. By default
+an added file is committed to the DVC cache. After `dvc add` dvc will
+`unprotect` all the data. For more information, visit
+[here](/doc/user-guide/update-tracked-file).
+
+Now, commit changes to `.dvc/config` and push them to your git remote:
+
+```dvc
+$ git add data.dvc .gitignore
+$ git commit . -m "add first version of the dataset"
+$ git tag -a "v1.0" -m "dataset v1.0"
+$ git push origin HEAD
+$ git push origin v1.0
+```
+
+Next, you can easily get this appear in your workspace by:
+
+```dvc
+$ cd /home/user/project/
+$ git pull
+$ dvc checkout
+```
+
+After `git pull`, you will be able to see a `data.dvc` file. To see more
+information on `.dvc` file format, visit
+[here](/doc/user-guide/dvc-file-format).
+
+`data` directory will now be a symbolic link to the NFS storage.
```

Comment:
worth writing something similar to the last paragraph in the introduction to reiterate on why links are important, worth showing an output of the `ls -a`